hs/Server.hs

```diff
@@ -23,6 +25,8 @@ import FacebookUtilities
 import Config (markdownPath, serverConf)
 import qualified Data.Text.Lazy.IO as LazyIO
 
+import Data.Int (Int64)
+
```

Comment:
@Ian-Stewart-Binks Ah, I will try to pay more attention to my use of spacing from now on.

I will make these corrections soon. Thanks for the hint.